sru-scripts/manual/azure-sru

```diff
@@ -326,5 +326,9 @@ for nics in "" "--nics $AZURE_NIC_NAME-$UBUNTU_RELEASE ${NIC_NAME}2-$UBUNTU_RELE
     ssh $sshopts $IP -- cloud-init status --wait --long
     echo 'After reboot'
     ssh $sshopts $IP -- "grep 'Update datasource' /var/log/cloud-init.log"
+
+    echo "Validate https://github.com/canonical/cloud-init/commit/22e683df"
+    echo "Azure needs to use __builtin__ agent by default on all releases"
+    ssh $sshopts $IP -- "egrep 'fabric' /var/log/cloud-init.log"
```

Comment:
I don know if I missing something, but shouldn't this check be applied on the `azure-20.2.45` report as well ? I could not find in the provided  log.

Reply:
@lucasmoura it may have been a stale representation in gihub.  You should see about 5 lines for each UBUNTU_RELEASE run like the following.

https://github.com/cloud-init/ubuntu-sru/pull/126/files#diff-92e4929cc7df391957b3985c9be81e93R1500-R1505

Reply:
Ah okay, I looked for the command in the shell script found on the `azure-sru-20.2.25` file, not directly to the output. Maybe we could add the command to the shell script as well, just to make it more consistent with the output result.